### DISCUSSION TITLE
Repositary

### DISCUSSION THREAD

**Discussion by CodeWithMe100** [community/community#33646] [New to GitHub] [answered]
How to change visibility of public repository to private?

**Comment by radar-blip** [comment]
Open you repo, go to settings, scroll down to the bottom and > Change Visibility

**Comment by Sagar-Sharma-7** [comment, marked as answer]
Hi @CodeWithMe100 
Repository > setting> scroll to down to danger zone> click on change visibility

**Comment by zarifpour** [comment]
Does [this](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/setting-repository-visibility#changing-a-repositorys-visibility) help?
